src/pages/login/loginStyle.tsx

```diff
@@ -0,0 +1,72 @@
+import { makeStyles } from "@material-ui/core/styles";
+const useStyles = makeStyles((theme) => ({
+  root: {
+    width: "100%",
+    height: "100vh",
+    display: "flex",
+    justifyContent: "center",
+    alignItems: "center",
+    backgroundImage: `linear-gradient(to top, #30cfd0 0%, #330867 100%)`,
+    "& .MuiContainer-root": {
+      backgroundColor: "#f1f2f4",
+      display: "flex",
+      flexDirection: "column",
+      alignItems: "center",
+      padding: "20px 0 40px 0",
+      borderRadius: "30px",
+    },
+    "& h2": {
+      marginBottom: "50px",
+      textAlign: "center",
+      fontSize: 35,
+      color: "#031054",
+      "&::after": {
+        content: '"-"',
+        display: "block",
+        height: "20px",
+        marginTop: "-20px",
+        color: "#02db8c",
+      },
+    },
+  },
+  textFieldRoot: {
+    marginBottom: "25px",
+    width: "90%",
+    marginLeft: "5%",
+    "& input": {
+      height: "30px",
+      backgroundColor: "transparent !important",
+    },
+    "& .MuiInputLabel-root": {
```

Comment:
you don't need to do these nestings. 
use InputLabelProps and InputProps 
https://material-ui.com/api/text-field/